doc/releases/release-notes-2.7.rst

```diff
@@ -166,18 +166,56 @@ Architectures
 Bluetooth
 *********
 
+* Updated all APIs and internal implementation to be conformant with the new
+  inclusive terminology in version 5.3 of the Bluetooth Core Specification
+
 * Audio
 
+  * Added the Microphone Input Control Service and client.
+  * Split the configuration options into connected and broadcast groups
+  * Added support for a new sent callback to be notified when an SDU has been
+    transmitted
+
+* Direction Finding
+
+  * Added configurability for conditional CTE RX support
+  * Added support for CTE periodic advertising chain transmissions
+  * Added support for connection-oriented CTE request and response
```

Comment:
Is this available upstream? isnt this using new LLCP work?

Reply:
No it is not, please remove it. That is implemented in about 30%...